tests/python_tests/test_kv_cache_eviction.py

```diff
@@ -190,3 +194,101 @@ def get_beam_search_seq_len_300() -> GenerationConfig:
 def test_dynamic_memory_allocation(tmp_path, params):
     run_cb_pipeline_with_ref(tmp_path, "facebook/opt-125m", scheduler_params=params[0], generation_config=params[1])
 
+
+@pytest.fixture(scope='module')
+def qwen2_converted_model(tmp_path_factory):
+    model_id = "Qwen/Qwen2-0.5B-Instruct"
+    model = OVModelForCausalLM.from_pretrained(model_id, export=True, trust_remote_code=True)
+    tokenizer = AutoTokenizer.from_pretrained(model_id)
+    models_path = tmp_path_factory.mktemp("cacheopt_test_models") / model_id
+    model.save_pretrained(models_path)
+    ov_tokenizer, ov_detokenizer = convert_tokenizer(tokenizer, with_detokenizer=True, skip_special_tokens=True)
+    serialize(ov_tokenizer, models_path / "openvino_tokenizer.xml")
+    serialize(ov_detokenizer, models_path / "openvino_detokenizer.xml")
+    qwen2_converted_model = ConvertedModel(model, tokenizer, models_path)
+    yield qwen2_converted_model
+    del qwen2_converted_model
+    del model
+
+
+@dataclass
+class LongBenchTestData:
+    subset: str
+    threshold: float
+    max_cache_usage_optimization_ratio: float
+    avg_cache_usage_optimization_ratio: float
+
+
+@pytest.mark.parametrize("test_struct", [
+    LongBenchTestData("samsum", 4, 1.6, 3.3),
+    LongBenchTestData("trec", 3.2, 2.0, 3.3),
+    LongBenchTestData("qasper", 5.8, 1.7, 3.6),
+])
+def test_optimized_generation_longbench(qwen2_converted_model, test_struct):
```

Comment:
is it explicitly not marked as `@pytest.mark.precommit` ? 
how long does this test take?

Reply:
Yes
On my local machine it takes 24 minutes, but on CI it takes about 60 minutes.

Reply:
Can we run this test in nightly jobs?

Reply:
yes, please add mark for `nightly`